Sources/SwiftInspectorVisitors/Tests/TypeSyntaxVisitorSpec.swift

```diff
@@ -31,68 +31,60 @@ final class TypeSyntaxVisitorSpec: QuickSpec {
   private var sut: TypeSyntaxVisitor!
 
   override func spec() {
-    describe("TypeProperties.merge(other:)") {
-      let type1 = TypeProperties(
-        name: "MyType",
-        properties: [
-          TypeProperties.PropertyData(
-            name: "thing",
-            typeAnnotation: "String",
-            comment: "",
-            modifiers: [.public, .instance])
-      ])
-      let type2 = TypeProperties(
-        name: "MyType",
-        properties: [
-          TypeProperties.PropertyData(
-            name: "foo",
-            typeAnnotation: "Int",
-            comment: "",
-            modifiers: [.public, .instance])
-      ])
-      let type3 = TypeProperties(
-        name: "AnotherType",
-        properties: [
-          TypeProperties.PropertyData(
-            name: "foo",
-            typeAnnotation: "Int",
-            comment: "",
-            modifiers: [.public, .instance])
-      ])
 
-      context("when both types have the same name") {
-        let result = try? type1.merge(with: type2)
-        it("succeeds") {
-          expect(result?.name) == "MyType"
+    describe("TypeSyntaxVisitor.merge(_:into:)") {
+
+      context("when there is no existing data about properties") {
+
+        context("and the new data is empty") {
+          it("returns the new data") {
+            let result = TypeSyntaxVisitor.merge([], into: nil)
+            expect(result).to(equal([]))
+          }
         }
 
-        it("has merged props") {
-          let set = Set(result?.properties ?? [])
-          let expectedSet: Set<TypeProperties.PropertyData> = [
-            TypeProperties.PropertyData(
+        context("and the new data describes one or more properties") {
+          context("returns the new data") {
+            let newPropertyData = PropertyData(
               name: "thing",
               typeAnnotation: "String",
               comment: "",
-              modifiers: [.public, .instance]),
-            TypeProperties.PropertyData(
+              modifiers: [.public, .instance])
+
+            let result = TypeSyntaxVisitor.merge([newPropertyData], into: nil)
+            expect(result).to(equal([newPropertyData]))
+          }
+        }
+      }
+
+      context("when there is existing data about properties") {
+
+        it("merges the existing and new data") {
+          let newPropertyData = PropertyData(
```

Comment:
nit: This setup is probably better places in a beforeEach block under the context?

Reply:
Since these are `let`s I didn't create a `beforeEach(...)` but I moved the values into the context. ad3e165